### ISSUE TITLE
[False Positive] aria-hidden-focus on elements with data-tabster-dummy

### ISSUE THREAD

**Issue by ultrababs** [microsoft/accessibility-insights-web#7598]
## Describe the bug

There is a [known issue](https://dev.azure.com/microsoftdesign/fluent-ui/_wiki/wikis/fluent-ui.wiki/334/Fluent-v9-known-issues?anchor=%60aria-hidden-focus%60-on-elements-with-%60data-tabster-dummy%60) with Fluent where they use Tabster which uses hidden focusable elements. These elements cause a numerous false positives for `aria-hidden-focus` on WCP accessibility scans. There have been quite a few open tickets about this over the years. Is there any possibility of hiding this false positive from WCP scans to prevent failures going forward?

## Screenshots

![Image](https://github.com/user-attachments/assets/ac7074d1-7700-4463-a238-fad25cf4039f)

![Image](https://github.com/user-attachments/assets/4b12d836-d141-4017-b6e9-140bf54ae3f2)

## Did you search for similar existing issues?

Yes, related tickets:
* https://github.com/microsoft/fluentui/issues/25133
* https://github.com/microsoft/fluentui/issues/30670
* https://github.com/microsoft/fluentui/issues/27056
* https://github.com/microsoft/fluentui/issues/25133
* https://github.com/microsoft/fluentui/issues/25133

Related Fluent teams channel threads:
* https://teams.microsoft.com/l/message/19:0201c9e5433b400b8eb01674ebd508b9@thread.skype/1746653580404?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ffe264f2-14d0-48b5-9384-64f808b81294&parentMessageId=1746653580404&teamName=Fluent%20Community&channelName=Accessibility&createdTime=1746653580404
* https://teams.microsoft.com/l/message/19:0201c9e5433b400b8eb01674ebd508b9@thread.skype/1745431404068?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ffe264f2-14d0-48b5-9384-64f808b81294&parentMessageId=1745431404068&teamName=Fluent%20Community&channelName=Accessibility&createdTime=1745431404068
* https://teams.microsoft.com/l/message/19:0201c9e5433b400b8eb01674ebd508b9@thread.skype/1740439619472?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ffe264f2-14d0-48b5-9384-64f808b81294&parentMessageId=1740439619472&teamName=Fluent%20Community&channelName=Accessibility&createdTime=1740439619472
* https://teams.microsoft.com/l/message/19:0201c9e5433b400b8eb01674ebd508b9@thread.skype/1730887445874?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ffe264f2-14d0-48b5-9384-64f808b81294&parentMessageId=1730887445874&teamName=Fluent%20Community&channelName=Accessibility&createdTime=1730887445874
* https://teams.microsoft.com/l/message/19:0201c9e5433b400b8eb01674ebd508b9@thread.skype/1715205641427?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ffe264f2-14d0-48b5-9384-64f808b81294&parentMessageId=1715205641427&teamName=Fluent%20Community&channelName=Accessibility&createdTime=1715205641427
... list goes on, too many to paste
